azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/models/CallCompositeCustomButtonOptions.java

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.calling.models;
+
+import com.azure.android.communication.ui.calling.CallCompositeEventHandler;
+
+/**
+ * Call composite custom button options.
+ */
+public final class CallCompositeCustomButtonOptions {
+
+    private final int drawableId;
+    private final String title;
+    private final CallCompositeEventHandler<CallCompositeCustomButtonClickEvent> onClickHandler;
+    private Boolean isEnabled = true;
```

Comment:
Why mix of immutable constructor, and mutable isEnabled?

If isEnabled can be changed later, shouldn't the rest also be updatable?

Reply:
In the constructor we take a must to provide parameters: icon, tittle and onclick. 
isEnabled is optional so we make it assignable via setter.